worldedit-core/src/main/java/com/sk89q/worldedit/command/tool/SelectionWand.java

```diff
@@ -29,36 +29,46 @@
 import com.sk89q.worldedit.internal.util.LogManagerCompat;
 import com.sk89q.worldedit.math.BlockVector3;
 import com.sk89q.worldedit.regions.RegionSelector;
+import com.sk89q.worldedit.util.Direction;
 import com.sk89q.worldedit.util.Location;
 import org.apache.logging.log4j.Logger;
 
+import javax.annotation.Nullable;
+
+//FAWE start - enum-ized
 public enum SelectionWand implements DoubleActionBlockTool {
     INSTANCE;
+//FAWE end
 
     private static final Logger LOGGER = LogManagerCompat.getLogger();
 
     @Override
-    public boolean actSecondary(Platform server, LocalConfiguration config, Player player, LocalSession session, Location clicked) {
+    public boolean actSecondary(Platform server, LocalConfiguration config, Player player, LocalSession session, Location clicked, @Nullable Direction face) {
```

Comment:
We should like... actually do something with this new direction face shizzle

Reply:
At some point, likely. Currently they do nothing but what upstream does (if even) but so we don't need to annotate them.